content/javascript/concepts/npm-in-javascript/npm-in-javascript.md

```diff
@@ -0,0 +1,43 @@
+---
+Title: "NPM in JavaScript"
+Subjects:
+  - "Bash/Shell"
+  - "Developer Tools"
+  - "Web Development"
+Tags: 
+  - "Bash/Shell"
```

Comment:
I kinda want to take "Bash/Shell" out of the Tags.md file because it's already in Subjects.